design_introspection-plugin/get_cmd.h

```diff
@@ -0,0 +1,36 @@
+#ifndef _GET_CMD_H_
+#define _GET_CMD_H_
+
+#include "kernel/register.h"
+
+USING_YOSYS_NAMESPACE
+
+struct GetCmd : public Pass {
+    using Filter = std::pair<std::string, std::string>;
+    using Filters = std::vector<Filter>;
+    using SelectionObjects = std::vector<std::string>;
```

Comment:
Shouldn't you be using RTLIL string in this vector, because we are going to just send this to the TCL list, rather than a C++ object?

Reply:
I don't really see a big benefit from using `RTLIL::IdString` in the current implementation. I would rather keep the dependency on the specific Yosys structs as low as possible.